.github/workflows/CI.yml

```diff
@@ -0,0 +1,129 @@
+            script: './scripts/run_test.sh'
+
+          - scala-version: "2.11.12"
+            BUILD: "base"
+            TEST_TARGET: "scalding-hadoop-test"
+            script: './scripts/run_test.sh'
+          - scala-version: "2.12.14"
+            BUILD: "base"
+            TEST_TARGET: "scalding-hadoop-test"
+            script: './scripts/run_test.sh'
+
+          - scala-version: "2.11.12"
+            BUILD: "base"
+            TEST_TARGET: "scalding-estimators-test"
+            script: './scripts/run_test.sh'
+          - scala-version: "2.12.14"
+            BUILD: "base"
+            TEST_TARGET: "scalding-estimators-test"
+            script: './scripts/run_test.sh'
+
+          - scala-version: "2.11.12"
+            BUILD: "base"
+            TEST_TARGET: "scalding-serialization scalding-spark"
+            script: './scripts/run_test.sh'
+          - scala-version: "2.12.14"
+            BUILD: "base"
+            TEST_TARGET: "scalding-serialization scalding-spark"
+            script: './scripts/run_test.sh'
+
+          - scala-version: "2.11.12"
+            BUILD: "base"
+            TEST_TARGET: "scalding-thrift-macros"
+            script: './scripts/run_test.sh'
+          - scala-version: "2.12.14"
+            BUILD: "base"
+            TEST_TARGET: "scalding-thrift-macros"
+            script: './scripts/run_test.sh'
+
+   #       - scala-version: "2.11.12"
+   #         BUILD: "test tutorials and matrix tutorials and repl"
+   #         script: './scripts/run_test.sh && ./scripts/build_assembly_no_test.sh scalding-assembly && ./scripts/test_tutorials.sh && ./scripts/build_assembly_no_test.sh scalding-assembly && ./scripts/test_matrix_tutorials.sh'
+   #       - scala-version: "2.12.14"
+   #         BUILD: "test tutorials and matrix tutorials and repl"
+   #         script: './scripts/run_test.sh && ./scripts/build_assembly_no_test.sh scalding-assembly && ./scripts/test_tutorials.sh && ./scripts/build_assembly_no_test.sh scalding-assembly && ./scripts/test_matrix_tutorials.sh'
+
+   #       - scala-version: "2.11.12"
+   #         BUILD: "test repl and typed tutorials and microsite"
+   #         script: './sbt ++$TRAVIS_SCALA_VERSION clean docs/makeMicrosite && ./scripts/build_assembly_no_test.sh scalding-repl && ./scripts/test_repl_tutorial.sh && ./scripts/build_assembly_no_test.sh scalding-core && ./scripts/test_typed_tutorials.sh && ./scripts/build_assembly_no_test.sh execution-tutorial && ./scripts/test_execution_tutorial.sh'
+   #       - scala-version: "2.12.14"
+   #         BUILD: "test repl and typed tutorials and microsite"
+   #         script: './sbt ++$TRAVIS_SCALA_VERSION clean docs/makeMicrosite && ./scripts/build_assembly_no_test.sh scalding-repl && ./scripts/test_repl_tutorial.sh && ./scripts/build_assembly_no_test.sh scalding-core && ./scripts/test_typed_tutorials.sh && ./scripts/build_assembly_no_test.sh execution-tutorial && ./scripts/test_execution_tutorial.sh'
```

Comment:
something about bash isn't escaping properly with the `&&` command chaining need to fix it.